components/inputs/test/input-time.test.js

```diff
@@ -113,11 +113,21 @@ describe('d2l-input-time', () => {
 			expect(elem.value).to.equal('0:00:00');
 		});
 
+		it('should apply custom default', async() => {
+			const elem = await fixture('<d2l-input-time label="label text" default-value="02:00:00"></d2l-input-time>');
+			expect(elem.value).to.equal('2:00:00');
+		});
```

Comment:
Can you add tests for the `startOfDay`, `endOfDay` and maybe an invalid value as well (which should throw)?